styles.css

```diff
@@ -371,7 +364,7 @@ body {
   background-image: url("images/contact-form.svg");
   background-position: right;
   background-size: contain;
-  background-repeat: no-repeat;
+  background-repeat: none;
```

Comment:
[OPTIONAL] Kindly make sure that the contact form of the mobile version is rendering correctly. The background is repeating. Please fix this :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/194717940-d0349208-0120-45b8-afe4-a5a4a0722845.png)